Sources/tart/OCI/Layerizer/DiskV2.swift

```diff
@@ -28,8 +29,18 @@ class DiskV2: Disk {
           let compressedData = try (data as NSData).compressed(using: .lz4) as Data
           let compressedDataDigest = Digest.hash(compressedData)
 
-          if try await !registry.blobExists(compressedDataDigest) {
-            _ = try await registry.pushBlob(fromData: compressedData, chunkSizeMb: chunkSizeMb, digest: compressedDataDigest)
+          try await retry(maxAttempts: 5, backoff: .exponentialWithFullJitter(baseDelay: .seconds(5), maxDelay: .seconds(60))) {
+            if try await !registry.blobExists(compressedDataDigest) {
+              _ = try await registry.pushBlob(fromData: compressedData, chunkSizeMb: chunkSizeMb, digest: compressedDataDigest)
+            }
+          } recoverFromFailure: { error in
+            if error is RuntimeError {
```

Comment:
Registry can also throw a `RegistryError`. I wonder if it makes sense to just narrow down the issue to a re-tryable HTTP/Network issue. For example, don't retry authorization. Or maybe retry all the `Error`s regadless. 🤔

Reply:
> I wonder if it makes sense to just narrow down the issue to a re-tryable HTTP/Network issue.

Implemented in https://github.com/cirruslabs/tart/pull/888/commits/4191fefe5c340cdade9bc96351387b6d8eee4122.